src/Microsoft.OData.Edm/EdmModelVisitor.cs

```diff
@@ -25,6 +25,11 @@ public void VisitEdmModel()
             this.ProcessModel(this.Model);
         }
 
+        public Task VisitEdmModelAsync()

```

Comment:
why not async ?

Reply:
@xuzhg This was recommended in some other PR to return task immediately instead of using `async/await` where possible especially if method perform one async. [Eliding `async/await`](https://dotnetdocs.ir/Post/48/eliding-async-await) is a technique that helps improve performance by overheads that come with `async/await`. The comment https://github.com/OData/odata.net/pull/3006#discussion_r1648585068